pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.526</version>
+        <version>1.598</version>
```

Comment:
Off the original version, I had problems running the test (on my Mac) which starts up a Jenkins instance because it had problems creating a socket.